packages/api/test/util/describeE2E.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017-2019 @polkadot/api authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+const WS_ENDPOINTS = {
+  'local': 'ws://127.0.0.1:9944/',
+  'substrate-master': 'ws://127.0.0.1:9945/',
+  'substrate-1.0': 'ws://127.0.0.1:9946/',
+  'substrate-2.0': 'ws://127.0.0.1:9947/',
+  'polkadot-master': 'ws://127.0.0.1:9948/',
+  'polkadot-alexander': 'ws://127.0.0.1:9949/',
+  'remote-polkadot-alexander': 'wss://poc3-rpc.polkadot.io/',
```

Comment:
Just be aware that unless we -

- setup a specific one, or
- use a service such as Polkasource or Unfrastructure

that this will not be possible anymore with Kusama for the Parity-hosted URLs. We get too much abuse which affects service to real users. 

Actual UI dev (hopefully) won't be affected, but I'll chat to the Parity folks around how that works, once it is setup and verified working. e2e,m unless tied to a specific IP though, will not be supportable.